### PR TITLE
Badge contrast

### PR DESCRIPTION
**Pull request by beatrizip**
Created a new variable that allows you to combine different backgrounds and color font.

### CODE REVIEW

File: components/atom/badge/src/index.scss [hunk 1]
```diff
@@ -10,7 +9,24 @@ $m-atom-badge-l: $m-m !default;
 $m-atom-badge-s: $m-s !default;
 $p-atom-badge: 0 $p-m !default;
 
-$tags: success $c-success, error $c-error, alert $c-alert, info $c-gray;
+$badges: (
+  success: (
+    background: $c-success,
```
Comment (jcger) [inline components/atom/badge/src/index.scss]: what about `bgc` instead of `background` and `c` instead of `font`?

**Review by jcger — APPROVED** [review]

File: components/atom/badge/src/index.scss [hunk 2]
```diff
@@ -50,22 +66,21 @@ $tags: success $c-success, error $c-error, alert $c-alert, info $c-gray;
...
+      fill: $c;
     }
 
     &-#{$type}--transparent {
       background-color: transparent;
-      color: $color;
-      fill: $color;
+      color: $bgc;
+      fill: $bgc;
+      padding: 0;
```
Comment (davidbarna) [inline components/atom/badge/src/index.scss:83]: can you tell us why you need padding 0 ?

**Review by davidbarna — APPROVED** [review]

File: components/atom/badge/src/index.scss (hunk 2 above)
Comment (beatrizip) [reply components/atom/badge/src/index.scss:83]: @davidbarna When it is not framed with a background, padding visually adds space. If you look at the dropbox paper there is no defined padding for transparent badges.

Comment (beatrizip) [reply components/atom/badge/src/index.scss:83]: image

**Pull request merged**